lib/skate/settings/user_settings.ex

```diff
@@ -34,7 +38,8 @@ defmodule Skate.Settings.UserSettings do
           user_id: user.id,
           # defaults, which won't get written if it exists
           ladder_page_vehicle_label: :run_id,
-          shuttle_page_vehicle_label: :vehicle_id
+          shuttle_page_vehicle_label: :vehicle_id,
+          vehicle_adherence_colors: :early_red
```

Comment:
A general concern I'd like to put in the back of your mind, rather than address now, is that we're developing a pattern of custom database types coupled to Elixir "types" with various boilerplate to translate. I think this is a good pattern and we should continue using it, but the boilerplate is starting to get on my nerves. Let's think sometime about how to refactor.

Reply:
Hmm, yeah, good point, it is kind of a lot of boilerplate. I don't have an immediate solution either, but I'll keep it in mind.